### DISCUSSION TITLE
How can we mirgate from Fabric to the latest Firebase crashlytics SDK in react native?

### DISCUSSION THREAD

**Discussion by AishuVaid** [invertase/react-native-firebase#4472] [Q&A] [answered]
Hi all. 
I have crashlytics implemented on my react native app. According to the email I received, there is a migration from Fabric to  Firebase Crashlytics SDKs and no crashes will be reported from the 15th of November 2020.
Can anyone please share how can I go about the migration for android as well as ios?
I couldn't find any official react native docs regarding this as yet?
@mikehardy Really looking forward to your help on this.
Thank you.

**Comment by mikehardy** [comment, marked as answer]
Just make sure you are up to date with react-native-firebase versions, we've got you covered https://github.com/invertase/react-native-firebase/blob/master/packages/crashlytics/CHANGELOG.md#800-2020-06-30

**Reply by andersonaddo** [reply]
Also, https://github.com/invertase/react-native-firebase/discussions/4410#discussioncomment-102881

**Reply by mikehardy** [reply]
@andersonaddo definitely true, thank you! I think the iOS dSYMS upload script is the one that trips up the most people now that we've got the test crash function working well (according to user reports). Don't forget to look at the iOS build scripts and switch the dSYMs uploader to the new script @AishuVaid 

Cheers

**Comment by AishuVaid** [comment]
@mikehardy Thanks a lot. Just one other thing. Are we expected to maintain the same version across all react-native-firebase libraries or are they all allowed to be different versions?

**Reply by mikehardy** [reply]
@AishuVaid the versions are different between the packages, but I try to use the current release of each package
